person.rb

```diff
@@ -1,11 +1,11 @@
-class Nameable
-  def correct_name
-    raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
-  end
-end
+require './nameable'
+require './capitalize_decorator'
+require './trimmer_decorator'
+require './base_decorator'
+require './rental'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
```

Comment:
- [ ] Please add Setters and Getters `attr_accessor` for @rentals, not Getters only `attr_reader` as you did so the person class can add new rentals.